features/sam_check.feature

```diff
@@ -26,8 +28,10 @@ Feature: Automatically checking a user's SAM status
   Scenario: Pending SAM check on DUNS edit
     Given I am a user without a verified SAM account
     And I am signed in
-    When I enter a new DUNS in my profile
+    When I visit my profile page
+    And I enter a new DUNS in my profile
     And I click on the "Submit" button
+
```

Comment:
yeah, feels weird to have the SAM warning in two places / slightly different. Add an issue?